sdk/eventgrid/azure-eventgrid/azure/eventgrid/_event_mappings.py

```diff
@@ -12,28 +12,28 @@ class SystemEventNames(str, Enum):
     visit https://docs.microsoft.com/azure/event-grid/system-topics.
     """
 
-    ACSChatMemberAddedToThreadWithUserEventName = (
+    AcsChatMemberAddedToThreadWithUserEventName = (
```

Comment:
Is is something public in the previous release?
naming change seems to be a breaking change to me, probably something worth to mention in the changelog.

If those are just added in this release, then I have no concern.

Reply:
i was on fence to add this in the changelog, but i agree -updated